cypress/objects/timeLogging.js

```diff
@@ -0,0 +1,38 @@
+class TimeLogging {
+    getUrl() {
+        return cy.url()
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title')
+    }
+
+    getCalendar() {
+        return cy.get('.calendar')
+    }
+
+    getTitleForm() {
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitles() {
+        return cy.get('.user-info__title')  
+    }
+
+    getTabsCount() {
+        return cy.get('.main-nav').find('li')
+    }
+
+    assertTodayDate() {
+        let today = new Date()
+        return cy.get('.calendar--today').find('.calendar__date').contains(today.getDate());
+    }
+
+    getTheColor() {
```

Comment:
Great function, but "get the color" of what? Let's specify that in the function name and call it something like `getActiveTabColor()` :)